### ISSUE TITLE
"file" not defined

### ISSUE THREAD

**Issue by ketch** [clawpack/geoclaw#234]
While testing the Python3 PR, I'm getting the following error:

```
  File "/home/travis/build/clawpack/clawpack/clawpack/clawutil/data.py", line 701, in write
    self.data_write(file, self.transverse_waves, 'transverse_waves')
NameError: name 'file' is not defined
```

Apparently `file` is a builtin (a type) in Python 2, but not in Python 3.  I'm not sure what the intent of this code is so I don't know how to make it Python 3 compatible.  It seems that `file` gets passed as the `name` argument to the `data_write` method.  I don't understand why a type would get passed as the name of something.  As far as I can tell, the value of `name` never actually gets used, so I've changed it to `None` here (which is also its default value).  Is that correct?

**Comment by mandli** [comment]
The `name` variable was there in case someone wanted to override the name in the data output file.  I imagine this is actually just a bug left over from an earlier version of the `ClawData` class as I cannot see why we wouldn't just want to write out the `transverse_wave` order here.  We should probably just change this to be
```
self.data_write('transverse_waves')
```

**Comment by rjleveque** [comment]
I think this was fixed in clawpack/clawutil@dd3c3f6cb
